src/ipfire.ipxe

```diff
@@ -9,7 +9,7 @@ goto ${menu} ||
 clear osversion
 set os IPFire
 menu ${os} - Image Sig Checks: [${img_sigs_enabled}]
-item 2.19-core118 ${space} ${os} 2.19 Core 118
+item 2.19-core118 ${space} ${os} 2.21 Core 122
```

Comment:
Make sure to change the second field too. (item)

Reply:
Thanks for the feedback.